pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1366,6 +1349,38 @@ func parseIpspec(ipspec *zconfig.Ipspec,
 		}
 		config.DnsServers = append(config.DnsServers, ds)
 	}
+	s := ipspec.GetSubnet()
+	if s == "" {
```

Comment:
Why do you need to return here? Shouldn't you parse the dhcp range?

Reply:
sure, done. I thought, without a subnet range, the DhcpRange does not make sense.